Quote generator/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+
+    <link rel="stylesheet" type="text/css"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./style.css">
+    <script src="./script.js"></script>
+</head>
+
+<body onload="startTime(); startDate()">
+    <div class="container">
+        <div id="date"></div>
+        <div id="display"></div>
+        <div id="content">
```

Comment:
You don't need the id on both _div_ and _p_ tags

Reply:
Id for p tag is given to change it's innerHTML and it is written in js.